content/schedule.md

```diff
@@ -20,3 +20,24 @@ weight: 4
 - Hackathon
   - Saturday, 20 April 2024, 9--18
 
+### Confirmed talks:
+
+In no particular order:
+
+1. My git-annex receipies for scaling (Sanders, Timothy)
+2. GIN-Tonic: (trying to) making datalad accessible to non-coders (Colomb, Julien; HU Berlin)
+3. Onedata as a Platform: Distributed Repository for Virtualizing Data and Long-term Preservation (Dutka, Lukasz; Onedata.org)
+4. Fusion of Multiple Open Source Applications for Data Management Workflows in Psychology and Neuroscience (Pfarr, Julia-Katharina; Philipps-University Marburg, CRC135 & The Adaptive Mind (TAM))
+5. Staying in Control of your Scientific Data with Git Annex (Büchau, Yann; Eberhard Karls Universität Tübingen)
+6. "git-annex is complete, right?" (Hess, Joey)
+7. Balancing Efficiency and Standardization for a Microscopic Image Repository on an HPC System (Thönnißen, Julia; Forschungszentrum Jülich, NFDI4BIOIMAGE)
+8. Optimisation in Network Engineering: Challenges and Solutions in Research Data Management (Breuer, Julius; Technische Universität Darmstadt, Chair of Fluid Systems)
+9. OpenNeuro and DataLad (Hardcastle, Nell; Stanford University)
+10. fMRI Pipelines on HPC with DataLad and ReproMan (Wexler, Joe; Stanford University)
+11. Workflow provenance-based scheduling (Martinez, Pedro; University of Calgary)
+12. Reproducible and replicable data science in the presence of patient  confidentiality concerns by utilizing git-annex and the Data Science Orchestrator (Brechtel, Markus Katharina; Research Group Cohorts in Infectious Disease and Cancer Research, University Hospitals Cologne and Frankfurt, German Centre for Infectious Diseases Research AND Kaluza, Philipp; Ghostroute Consulting)
+13. Neuroscientific data management using DataLad (Kosciessa, Julian; Donders Institute for Brain, Cognition and Behaviour, Radboud University)
+14. datalad-annex: a git-remote helper to deposit a repository in any place reachable by a git-annex special remote implementation (Hanke, Michael; Forschungszentrum Jülich)
+15. datalad-registry: a registry of datalad datasets with metadata extracted (To, Isaac; Dartmouth College, Center for Open Neuroscience)
```

Comment:
What about "datalad-registry: a registry of datalad datasets with extracted metadata" or simply "datalad-registry: a registry of datalad datasets with metadata"

Reply:
make it as a github suggestion: use 
![image](https://github.com/distribits/distribits-2024-website/assets/39889/c20f4884-3d94-42f0-8e52-fec33b86861d)